frontend/packages/dev-console/src/components/monitoring/overview/MonitoringTab.tsx

```diff
@@ -1,13 +1,29 @@
 import * as React from 'react';
 import { OverviewItem } from '@console/shared';
-import MonitoringOverview from './MonitoringOverview';
+import { PodModel } from '@console/internal/models';
+import { Firehose } from '@console/internal/components/utils';
+import MonitoringOverviewWrapper from './MonitoringOverviewWrapper';
 
 type MonitoringTabProps = {
   item: OverviewItem;
 };
 
-const MonitoringTab: React.FC<MonitoringTabProps> = ({ item: { obj: res, events } }) => {
-  return <MonitoringOverview resource={res} events={events} />;
+const MonitoringTab: React.FC<MonitoringTabProps> = ({ item }) => {
+  const resources = [
+    {
+      isList: true,
+      kind: 'Event',
+      namespace: item?.obj?.metadata?.namespace,
+      prop: 'events',
+      fieldSelector: `involvedObject.kind=${PodModel.kind}`,
+    },
+  ];
```

Comment:
I think we agreed that the events in the monitoring tab should be the same events shown in the workloads events tab and not the individual pods associated with the workload.
 
cc @sspeiche @abhi-kn